### PR TITLE
Refactor getDOM to fix issue when running Ember in FastBoot with Glimmer

### LINKED ISSUE #31: Error getting dom in FastBoot using Glimmer 2
Hey there. 

I'm getting an error when rendering mobiledoc in FastBoot with Ember >= 2.10 (which includes Glimmer 2). 

```
There was an error running your app in fastboot. More info about the error:
 TypeError: Cannot read property 'document' of undefined
    at Class._renderMobiledoc (/Users/josemarluedke/Projects/Temp/mobiledoc-fastboot/dist/fastboot/vendor.js:93572:26)
    at Class.didReceiveAttrs (/Users/josemarluedke/Projects/Temp/mobiledoc-fastboot/dist/fastboot/vendor.js:93545:12)
```

Reproduction app is here: https://github.com/josemarluedke/mobiledoc-fastboot

### PR DESCRIPTION
**Pull request by josemarluedke**
This fixes #31.

`getDOM` was extracted from https://github.com/yapplabs/ember-wormhole/blob/master/addon/utils/dom.js#L48.

### CODE REVIEW

File: addon/components/render-mobiledoc.js [hunk 1]
```diff
@@ -99,8 +99,7 @@ export default Ember.Component.extend({
   }),
 
   _renderMobiledoc() {
-    let domHelper = getDOM(this);
-    let dom = domHelper.document;
+    let dom = getDOM(this);
```
Comment (mixonic) [inline addon/components/render-mobiledoc.js]: `getDOM` seems to be a misnomer with your refactor. The `document` is being returned. Perhaps this API should be renamed `getDocument`?

**Comment by mixonic** [comment]
@josemarluedke can you also add Ember 2.5 and 2.9 to https://github.com/bustlelabs/ember-mobiledoc-dom-renderer/blob/master/config/ember-try.js? This would ensure we're testing all those funky code paths in the fallback document stuff.

Thanks for this excellent patch! 👏

**Comment by josemarluedke** [comment]
@mixonic PR updated!

**Comment by mixonic** [comment]
Thanks @josemarluedke :-D

**Pull request merged**